### DISCUSSION TITLE
Launch Gunicorn via Poetry

### DISCUSSION THREAD

**Discussion by rosaLux161** [python-poetry/poetry#8482] [Q&A]
Hi,

I would like to start Gunicorn through Poetry. So far I have in the scripts section of pyproject.toml the following:

```
[tool.poetry.scripts]
my-super-duper-package = 'my_super_duper_package.my_super_duper_package:start'
```

This starts the flask server via the start function:

```
def start():
    app.run(host='0.0.0.0', port=1234, debug=True)
```

But in the scripts section I can only specify a jump-in point, is there a way to also store a command like:

`gunicorn -b 0.0.0.0:1234 -c ./gunicorn_conf.py my_super_duper_package:app`

This could start the Flask server including Gunicorn directly with a command in the console and would encapsulate the whole package even more.

This could be interesting, but I don't think it's what I'm looking for: [2310](https://github.com/python-poetry/poetry/issues/2310#)

**Comment by radoering** [comment]
I suppose you are looking for a task runner, e.g. https://github.com/nat-n/poethepoet.

**Reply by rosaLux161** [reply]
But for this I have to install Poetry to run this app. With the script section, I can just install it on any device and just run the app calling the app name.

**Reply by radoering** [reply]
Then, maybe just [subprocess](https://docs.python.org/3/library/subprocess.html)?

**Reply by dimbleby** [reply]
you started with "I would like to start Gunicorn through Poetry" and then immediately pivoted to "but for this I would have to install Poetry".  What do you actually want?

perhaps you should just write a one-line shell script to run the command you want to run.
